src/index.ts

```diff
@@ -303,72 +280,22 @@ class Envoy {
         let adminResponse;
         // - Install App - This admin function creates cells for each dna with associated nick, under the hood.
         try {
-          log.info("Installing App with HHA ID (%s) as Installed App ID (%s) ", hha_hash, installed_app_id);
-
-					const installDna = dna => {
-						const timeout = 5_000;
-						const happFilePath = tmp.tmpNameSync();    
-						const file = fs.createWriteStream(happFilePath);
-						const urlObj = new URL(dna.src_path);
-						urlObj.protocol = "https";
-						const downloadDnaPath = new Promise(resolve => {
-							requestUrl({
-									uri: urlObj.toString()
-							})
-							.pipe(file)
-							.on('finish', () => {
-									log.debug(`Downloaded file from ${urlObj.toString()} to ${happFilePath}`);
-									const dnaInfo = { nick: dna.nick, path: happFilePath };
-									resolve(dnaInfo);
-							})
-							.on('error', (error) => {
-								// Delete the file async if error occurs
-								fs.unlink(happFilePath, () => {
-									log.debug(`Failed to download file ${urlObj.toString()} for ${dna.nick}`);
-									throw new Error(error.message);
-								});
-							})
-						});
-
-						const timeoutCheck = new Promise((resolve, reject) => {
-							let id = setTimeout(() => {
-								clearTimeout(id);
-								reject('Timed out in '+ timeout + 'ms.')
-							}, timeout)
-						});
-
-						return Promise.race([
-							downloadDnaPath,
-							timeoutCheck
-						]);
-					};
-
-          let dnas;
-          if (this.opts.hosted_app && this.opts.hosted_app!.dnas && this.opts.mode === Envoy.DEVELOP_MODE) {
-						if (this.opts.hosted_app!.usingURL) {
-							const installedDnas = await promiseMap(this.opts.hosted_app.dnas, installDna);
-							log.debug('installedDnas : %s', installedDnas);
-							
-							dnas = installedDnas;
-						} else {
-							dnas = this.opts.hosted_app.dnas;
-						}
-          }
-
-					adminResponse = await this.callConductor("master", 'installApp', {
-            installed_app_id,
+          log.info("Installing App with HHA ID (%s) as Installed App ID (%s) ", hha_hash, hosted_instance_app_id);
+          let a = appInfo.cell_data.map(([cell_id, dna_alias]) => ({ nick: dna_alias, hash: cell_id[0] }))
+  				adminResponse = await this.callConductor("admin", 'installApp', {
+            installed_app_id: hosted_instance_app_id,
             agent_key: buffer_agent_id,
-            dnas: dnas || await promiseMap(app.happ_bundle.dnas, installDna),
```

Comment:
Why are we removing the awareness of the `opts.hosted_app.dnas`?  This value would allow us to test different dnas quickly.